addons/hr_holidays/models/hr_leave.py

```diff
@@ -1586,7 +1586,7 @@ def _get_responsible_for_approval(self):
             if self.holiday_status_id.responsible_ids:
                 responsible = self.holiday_status_id.responsible_ids
             else:
```

Comment:
This is not needed, because if there is no responsible_ids, then we should get default `responsible = self.env.user`